srcopsmetrics/entities/tools/knowledge.py

```diff
@@ -133,3 +133,25 @@ def get_interactions(comments) -> Dict:
             # we count by the num of words in comment
             interactions[comment.user.login] += len(comment.body.split(" "))
         return interactions
+
+    @staticmethod
+    def get_labels(issue: Issue) -> Dict[str, Dict[str, Union[int, str]]]:
+        """Get non standalone labels by filtering them from all of the labels."""
+        labels: Dict[str, Dict[str, Union[int, str]]] = {}
+
+        for event in issue.get_timeline():
+
+            if event.event != "labeled":
+                continue
+
+            label = event.__dict__.get("_rawData")["label"]
+            if label["name"] in labels.keys():
```

Comment:
what is this condition doing?

Reply:
it's in case of duplicate labels. Let's consider the scenario:
PR was labeled with X
PR was unalabeled with X
PR was labeled with X

this sounds like a contributor mistake I would not want to count it, therefore i ignore any other `labeled` event with the same label name